tools/canary_audit/tests/test_schema_integration.py

```
@@ -0,0 +1,118 @@
+		self.assertEqual(with_stale_waiver.reference_report["summary"]["staleWaiverCount"], 1)
+
+	@staticmethod
+	def _write(root: Path, relative: str, content: str | bytes) -> None:
+		path = root / relative
+		path.parent.mkdir(parents=True, exist_ok=True)
+		if isinstance(content, bytes):
+			path.write_bytes(content)
+		else:
+			path.write_text(content, encoding="utf-8")
+
+	classmethod
```

Ruff (0.15.20)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)